### PR TITLE
Google search result api testcases

### PR DESCRIPTION
**Pull request by nidhiya-pious**

### CODE REVIEW

File: cypress/e2e/GoogleSearchResult/home.cy.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        cy.componentVisiblityCheck('#documentation-tab', 'Documentation');
+    });
+
+    it('10. Test to check "Related Products" container is present', () => {
+        cy.componentVisiblityCheck('.h1', 'Related Products');
+    });
+
+    it('11. Test to check footer is present', () => {
+        cy.componentVisiblityCheck('footer.bg-primary');
+    });
```
Comment (sinchanagoudar) [inline cypress/e2e/GoogleSearchResult/home.cy.js:52]: Missing the test case for check the hyper link  **check the documentation.**
image

File: cypress/e2e/GoogleSearchResult/apiInfo.cy.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    it('7. Test to check "Code example" windows are present', () => {
+        cy.get('img[alt="Google Search Results API"]').should('be.visible');
+    });
+
+    it('8. Test to check "Related Articles" subtitle and artcle container are present', () => {
+        cy.checkTextVisibility('Related Articles');
+        cy.componentVisiblityCheck('.card-deck');
+    });
+});
```
Comment (sinchanagoudar) [inline cypress/e2e/GoogleSearchResult/apiInfo.cy.js:54]: Test cases is missing for these titlts 
image

File: cypress/e2e/GoogleSearchResult/documentation.cy.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    it('1. Test to "Accept Cookies" if not already accepted', () => {
+        cy.AcceptCookies();
+    });
+  
+    it('2. Test to check "Documentation" tab visibility', () => {
+        cy.componentVisiblityCheck('#documentation-tab');
+    });
+  
+    it('3. Test to click "Documentation" tab', () => {
+        cy.get('#documentation-tab').click();
```
Comment (NaveenK15) [inline cypress/e2e/GoogleSearchResult/documentation.cy.js:18]: add apirequest api interupt here too

Comment (sinchanagoudar) [reply cypress/e2e/GoogleSearchResult/documentation.cy.js:17]: Add API interupt wait here.
Also the tab has an id use that to get the element

**Pull request merged**